mentoring/mentor-guide.md

```diff
@@ -0,0 +1,101 @@
+Mentor Guidelines
+=======
+*WIP*
+
+With over 37 special interest and working groups (SIGs/WGs), the Kubernetes organization is very large to navigate through and no one has all of the answers. Contributors rely on our docs, commenting on PRs/Issues, finding a mentor on their own, and any time they can interact with others to get up to speed. Whether it’s Slack, update meetings, or KubeCons; contributors want to discuss technical specifics, the landscape, or how to be a valuable member. Being a mentor can help accelerate someone's path to being more productive in our ecosystem.
+
+There are many ways you can be a mentor within the Kubernetes ecosystem. Listening and being generally helpful is an informal start but we’ve drawn together some programs that will be impactful. These programs also take your time into consideration for both parties - we know that it’s difficult to carve out steady time to mentor someone. Bonus: we set up the communication framework for you already so your expectations are set when you are ready to dig into mentoring.
```

Comment:
`informal start,`*

Reply:
`These programs also take your time into consideration for both parties` --> `These programs take both the mentee's AND the mentor's into consideration`

Reply:
` - we know that it’s difficult to carve out steady time to mentor someone` --> ` We know that it’s difficult to carve out steady time to mentor someone, and we appreciate that contribution to the Kubernetes community!`

Reply:
Also, not sure sure that the bonus is really a bonus; more so just a facet of the program(s).

Reply:
ty!